src/frontend/src/components/ProposedSolutionView.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { ProposedSolution } from 'shared';
+
+import PageBlock from '../layouts/PageBlock';
+import { Badge, Col, Container, Row } from 'react-bootstrap';
+import { dollarsPipe, weeksPipe } from '../utils/Pipes';
+import { useTheme } from '../hooks/Theme.hooks';
+
+interface ProposedSolutionViewProps {
+  proposedSolution: ProposedSolution;
+  selected: boolean;
+  setter: any;
+}
+
+const ProposedSolutionView: React.FC<ProposedSolutionViewProps> = ({
```

Comment:
Now that I've thought ab it, can you rename this ProposedSolutionButton or ProposedSolutionSelectItem (or smthn else if you think of something better) because that's more accurate to how it's being used (imo you don't usually click views). Also, we wanna keep the name distinct from the ProposedSolutionView being used to display the proposed solutions on the CR detail page. Can you also move this file to the ChangeRequestDetailPage folder, I don't think it has enough general applications to be a component.

Reply:
Changed to ProposedSolutionSelectItem and moved it to the crdp folder